packages/page-header/PageHeader.js

```diff
@@ -89,7 +90,7 @@ const PageHeader = ({
   return (
     <>
       <div className="d-flex align-items-start flex-shrink-0">
-        {React.isValidElement(crumbs) ? (
+        {showCrumbs && React.isValidElement(crumbs) ? (
```

Comment:
Wouldn't this only apply when `crumbs` is passed as a React node?

Reply:
updated